### PR TITLE
Closed #71 @Brauliodiez @JTrillo

### LINKED ISSUE #71: Resend to home when user just uses a deep link
A link like:

https://www.englishquiz.fun/app.html#/test-fill-gap

Or to any other test, could make the app look like broken, when that happens redirect to main page.

### PR DESCRIPTION
**Pull request by vandralca**
Closed #71 @Brauliodiez @JTrillo

### CODE REVIEW

**Comment by JTrillo** [comment]
Right now, final score component is unreachable, since is accessed through navbar and tests components. I'm gonna fix that quickly.

**Comment by JTrillo** [comment]
I've included **fromTest** check in order to reach final score component from tests or navbar.

**Review by JTrillo — APPROVED** [review]
Everything is ok. With this approach. issue #72 gets also solved and PR #73 discarded

File: src/core/router/router.component.tsx [hunk 1]
```diff
@@ -18,27 +38,27 @@ export const RouterComponent: React.FunctionComponent = () => {
...
           exact={true}
           path={switchRoutes.configureVerbs}
-          component={ConfigureVerbsScene}
+          render={location => handleRedirect(ConfigureVerbsScene, location)}
         />
         <Route
           exact={true}
           path={switchRoutes.finalScore}
-          component={FinalScoreScene}
+          render={location => handleRedirect(FinalScoreScene, location, 'fromTest')}
```
Comment (brauliodiez) [inline src/core/router/router.component.tsx:46]: Mmm... I think we can just use an interceptor and avoid having to code the redirecto on each route.

Some tips (in this case this is applied to Auth, but we can use a similar approach):

https://lemoncode.net/lemoncode-blog/2020/12/20/react-router-y-rutas-autorizadas-i

Then, a possible simple way to check if we need to return home, we could simply check, _verbsContext_ if the verbsContext array is null, or empty then jump back to home page.

What do you think?

Comment (JTrillo) [reply src/core/router/router.component.tsx:46]: Please check PR #77. I though that is better to separate both proposals till we choose one of them.

Comment (brauliodiez) [reply src/core/router/router.component.tsx:46]: Finally we will for for PR #77, thanks a lot Vicente and Joaquin it has been great working on this and coming into a final clean solution, we will go through it on next Thursday session

**Pull request closed without merging**